compiler/ployfills/es5/ployfill.js

```diff
@@ -0,0 +1,5 @@
+import { mark, wrap } from 'https://esm.sh/regenerator-runtime@0.13.7'
+
+if (!('regeneratorRuntime' in window)) {
+  window.regeneratorRuntime = { mark, wrap }
+}
```

Comment:
btw I think the correct spelling is `polyfill`.

Reply:
I couldn't find any code that actually references this folder so, assuming that this functionality will be added later on when bundling is added, I just renamed the folder and the file name.